cloud/tasks/logging/journald_logger.go

```diff
@@ -42,23 +47,100 @@ func getPriority(level Level) journal.Priority {
 
 ////////////////////////////////////////////////////////////////////////////////
 
-func getFields(fields ...log.Field) map[string]string {
+type rawField struct {
```

Comment:
I do not see how is the extraction of the getField() method helps here? I do not see any other usage of the method. I think we should keep the code as it was working previously.

Reply:
Sorry, didn't notice the usage of the rawField